src/lib.rs

```diff
@@ -44,11 +44,13 @@ extern crate serde_crate as serde;
 #[cfg(feature = "serde")]
 extern crate serde_with;
 
-pub mod schema;
+mod schema;
 mod create;
 mod asset;
 mod transitions;
 
 pub use asset::{Asset, Error};
 pub use create::Rgb20;
-pub use schema::{schema, subschema, SCHEMA_ID_BECH32, SUBSCHEMA_ID_BECH32};
+pub use schema::{
+    Rgb20Schemata, BECH32_SCHEMA_ID_INFLATIONARY, BECH32_SCHEMA_ID_ROOT, BECH32_SCHEMA_ID_SIMPLE,
+};
```

Comment:
Since the `schema` module is now private, I think we need to re-export `FieldType` and `OwnedRightType` that, for example, are useful for reading metadata or to create custom outputs